src/clj/rems/db/form.clj

```diff
@@ -48,17 +50,46 @@
   (or (form-in-use-error form-id)
       {:success true}))
 
-(defn- generate-field-ids [fields]
-  (map-indexed (fn [index field]
-                 (assoc field :field/id (inc index)))
-               fields))
+(defn validate-given-ids [fields]
```

Comment:
I'm kinda missing a docstring for this, even though the test makes the behaviour clear